nltk/data.py

```diff
@@ -335,7 +346,7 @@ class BufferedGzipFile(GzipFile):
     ``BufferedGzipFile`` is useful for loading large gzipped pickle objects
     as well as writing large encoded feature files for classifier training.
     """
-    SIZE = 2 * 2**20
+    SIZE = 2 * 2 ** 20
```

Comment:
I kind of liked `2 * 2**20` more because operation priorities are more clear this way. Maybe it worths it to add a variable:

```
MB = 2 ** 20
SIZE = 2 * MB
```